imports/plugins/core/orders/client/templates/workflow/cancelOrder.js

```diff
@@ -0,0 +1,108 @@
+import { Meteor } from "meteor/meteor";
+import { Template } from "meteor/templating";
+
+const validateComment = (comment) => {
```

Comment:
is it possible to use a set of preset comments that the admin can select from and use rather than allowing them to enter them themselves thereby eliminating the need to validate comments